From 2014d037d3488f1a53e1c0c83263e77322ac14ef Mon Sep 17 00:00:00 2001
From: Peter Bergin <peter@berginkonsult.se>
Date: Mon, 19 Sep 2022 17:21:28 +0200
Subject: [PATCH] libtraceevent: make it possible to set libdir and mandir from
 outside

When building in build frameworks such as openembedded it is desired
to use the defintion of libdir and mandir from the build framework.

Upstream-Status: Submitted [https://lore.kernel.org/linux-trace-devel/20221031212140.170601-1-peter@berginkonsult.se]
Signed-off-by: Peter Bergin <peter@berginkonsult.se>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 194bbdd..aa1984d 100644
--- a/Makefile
+++ b/Makefile
@@ -45,8 +45,8 @@ endif
 
 libdir_relative ?= $(libdir_relative_temp)
 prefix ?= /usr/local
-libdir = $(prefix)/$(libdir_relative)
-man_dir = $(prefix)/share/man
+libdir ?= $(prefix)/$(libdir_relative)
+man_dir ?= $(prefix)/share/man
 man_dir_SQ = '$(subst ','\'',$(man_dir))'
 pkgconfig_dir ?= $(word 1,$(shell $(PKG_CONFIG) 		\
 			--variable pc_path pkg-config | tr ":" " "))
-- 
2.34.1

